launcher/InstanceImportTask.cpp

```diff
@@ -558,9 +558,13 @@ void InstanceImportTask::processModrinth()
                 // Do not use requireUrl, which uses StrictMode, instead use QUrl's default TolerantMode
                 // (as Modrinth seems to incorrectly handle spaces)
                 file.download = Json::requireString(Json::ensureArray(obj, "downloads").first(), "Download URL for " + file.path);
-                if (!file.download.isValid() || !Modrinth::validateDownloadUrl(file.download)) {
-                    throw JSONValidationError("Download URL for " + file.path + " is not a correctly formatted URL");
-                }
+
+                if(!file.download.isValid())
+                    throw JSONValidationError(tr("Download URL for %1 is not a correctly formatted URL").arg(file.path));
+                else if(!Modrinth::validateDownloadUrl(file.download))
+                    throw JSONValidationError(
+                            tr("Download URL for %1 is from a non-whitelisted by Modrinth domain: %2").arg(file.path, file.download.host()));
```

Comment:
Exception descriptions are for the developers to read and understand, not to be translated into the user's language so we don't know what the exception was. The "missing" tr() is by design. This is why nothing in the debug logs is translated either.

Reply:
This shows in the error dialog shown to the user

Reply:
> This shows in the error dialog shown to the user

That makes sense to translate then, maybe it should be accompanied by an untranslated description of the error in qDebug?

Reply:
Sure!